src/config.ml

```diff
@@ -1042,22 +1017,22 @@ let inline_payload element =
   Cstruct.(sub cs 0 64, sub cs 64 64, sub cs 128 64, sub cs (128 + 64) 64)
 
 let a_tls_crypt_v2_client_payload force_cookie =
-  ( inline_pem_payload tls_crypt_v2_client_pem_name "tls-crypt-v2" >>= fun cs ->
-    if Cstruct.length cs >= 256 + Mirage_crypto.Hash.SHA256.digest_size then
-      return cs
-    else fail "bad tls-crypt-v2 client key size" )
-  >>| fun cs ->
-  let key = Cstruct.(sub cs 0 64, sub cs 64 64, sub cs 128 64, sub cs 192 64) in
-  let wkc = Cstruct.shift cs 256 in
-  B (Tls_crypt_v2_client, (key, wkc, force_cookie))
+  let line = take_till (( = ) '\n') <* char '\n' in
```

Comment:
I missed that before, but maybe we should use `a_newline`:
```suggestion
  let line = take_till (function '\n' | '\r' -> true | _ -> false) <* a_newline in
```
The reason is to support `\r\n` newlines.